Buildings/Fluid/HeatExchangers/Examples/WetCoilCounterFlowPControlAutoTuning.mo

```diff
@@ -0,0 +1,215 @@
+within Buildings.Fluid.HeatExchangers.Examples;
+model WetCoilCounterFlowPControlAutoTuning
```

Comment:
It seems that the controller did not control the measured value to the set point:
![WetCoilCounterFlowPControlAutoTuning](https://github.com/lbl-srg/modelica-buildings/assets/23491921/bb3241a7-0bc1-406b-9396-dc1366b63d3d)